lib/File/Find/Rule.pm

```diff
@@ -485,6 +485,18 @@ sub relative () {
     $self;
 }
 
+=item C<normalize>
+
+Normalize paths found using C<File::Spec->canonpath>
```

Comment:
A little more wording about what a path being "normalised" means would help.  One that doesn't force someone to try and understand it from File::Spec's documentation, as it's actually fairly bad http://search.cpan.org/~rjbs/PathTools-3.60/lib/File/Spec.pm#METHODS

And should this option be called canonise rather than normalize, assuming that File::Spec->canonpath is a well-known behaviour?